lib/converters/v1.0.0/converter-v1-to-v21.js

```diff
@@ -52,8 +53,6 @@ _.assign(Builders.prototype, {
 
         var auth = Builders.super_.prototype.auth.call(this, entityV1);
 
-        if (auth && (auth.type === 'noauth')) { return { type: 'noauth' }; }
```

Comment:
Why is this removed?

Reply:
This is now being handled in the util call below.